nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -35,7 +35,7 @@ with lib;
   # Enable sound in graphical iso's.
   hardware.pulseaudio.enable = true;
 
-  environment.systemPackages = [
+  environment.defaultPackages = [
```

Comment:
this would remove nano, perl (might be needed for some nixos tools but no issue noticed), rsync (people might wart to use to sync their config.)

`systemPackages` might be fine here

Reply:
Tested installation with the change and appears to work without issue

Reply:
yes. but people who would like to manually install might want to have nano and rsync

Reply:
I wanted to remove all of those packages for faster build time. It was quite hard to remove these when they are part of `systemPackages` (which _also_ includes critical packages).

Reply:
Just directly adding nano and rsync to the defaultPackages should be then right?

Reply:
Added to defaultPackages